src/motor/digital/servo/ServoMotor.hpp

```diff
@@ -9,7 +9,7 @@
 #pragma once
 
 #include <Servo.h>
-#include <stdint.h> // NOLINT(modernize-deprecated-headers)
+#include <stdint.h>
```

Comment:
:warning: **modernize-deprecated-headers** :warning:
inclusion of deprecated C++ header 'stdint.h'; consider using 'cstdint' instead
```suggestion
#include <cstdint>```